### ISSUE TITLE
panic: interface conversion: interface {} is nil, not float64

### ISSUE THREAD

**Issue by GoldstarZV** [o-fl0w/stash-vr#23]
I'm getting this error when I launch my container @o-fl0w 

```
Nov 18, 06:25:15 INF config="{StashGraphQLUrl:REDACTED(33) StashApiKey:REDACTED(143) FavoriteTag:FAVORITE Filters: IsSyncMarkersAllowed:true LogLevel:info IsRedactDisabled:false ForceHTTPS:false IsHeatmapDisabl
htPx:0 IsPlayCountDisabled:false StimhubUrl:}" mod=default
Nov 18, 06:25:15 INF Stash-VR version=ofl0w/stash-vr:v0.7.3 mod=default
Nov 18, 06:25:15 INF Stash version=v0.23.1-36-g72779e61 mod=default

panic: interface conversion: interface {} is nil, not float64

goroutine 19 [running]:
stash-vr/internal/stash.FindSavedFilterIdsByFrontPage({0x99c1f8, 0xc000208000}, {0x998840?, 0xc00020c0f0?})
    /build/internal/stash/filter.go:57 +0x44d
stash-vr/internal/sections/internal.SectionsByFrontpage({0x99c1f8?, 0xc000208000}, {0x998840?, 0xc00020c0f0}, {0x0, 0x0})
    /build/internal/sections/internal/frontpage.go:12 +0x51
stash-vr/internal/sections.build.func1()
    /build/internal/sections/sections.go:44 +0x8c
created by stash-vr/internal/sections.build in goroutine 1
    /build/internal/sections/sections.go:42 +0x1d9
```

I had stash-vr set up properly before, but with this latest image it doesn't seem to work 

On the latest version of stash on develop branch v0.23.1-36-g72779e61, and I definitely did supply the API key

**Comment by GoldstarZV** [comment]
ok I fixed it, it was because stash-vr went looking for saved filters in my stash front page, but because I didn't add any of my own saved filters in the front page, it failed on that. I added a saved filter to the front page and now it's working, you probably should add a nil check or something so it doesn't panic

**Comment by o-fl0w** [comment]
Seems for some reason saved filters in stash can get a nil id, perhaps during migration? Thank for reporting this, added a fix for it in v0.7.4 https://github.com/o-fl0w/stash-vr/commit/759505e1a819f45da6e3b1778048abaf16af3d37
